### PR TITLE
[DOC] Improve readme

### PR DESCRIPTION
**Pull request by csouchet**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+You are here to help on `mxgraph-sketch`? Awesome, feel welcome and read the following guidelines in order to know how to contribute, to ask questions and to make `mxgraph-sketch` such a great tool.
+
+All members of our community are expected to follow our [Code of Conduct](https://github.com/process-analytics/.github/blob/main/CODE_OF_CONDUCT.md). Please make sure you are welcoming and friendly in all of our spaces.
+
+## Contributions 
+
+There are many ways to contribute:
+
+- help people with the questions they ask on the [Github Issues](https://github.com/process-analytics/mxgraph-sketch-js/issues)
```
Comment (tbouffard) [inline CONTRIBUTING.md]:
```suggestion
- help people with the questions they ask on the [Github Issues](https://github.com/process-analytics/mxgraph-sketch/issues)
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+- submitting bug reports and feature requests in the [Github Issues](https://github.com/process-analytics/mxgraph-sketch-js/issues/new)
```
Comment (tbouffard) [inline CONTRIBUTING.md]:
```suggestion
- submitting bug reports and feature requests in the [Github Issues](https://github.com/process-analytics/mxgraph-sketch/issues/new)
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+- writing code which can be incorporated into `mxgraph-sketch` itself
+- improving the documentation
+
+## Code and documentation changes guidelines
+
+For all contributions, please respect the following guidelines:
+
+1. If you've noticed a bug or have a feature request, let us know in the [GitHub Issue tracker](https://github.com/process-analytics/mxgraph-sketch-js/issues/new )! So we can confirm the bug or approve your feature, and provide feedback, before starting to code :slightly_smiling_face:
```
Comment (tbouffard) [inline CONTRIBUTING.md]:
```suggestion
1. If you've noticed a bug or have a feature request, let us know in the [GitHub Issue tracker](https://github.com/process-analytics/mxgraph-sketch/issues/new )! So we can confirm the bug or approve your feature, and provide feedback, before starting to code :slightly_smiling_face:
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+[Fork mxgraph-sketch](https://help.github.com/articles/fork-a-repo) and create a branch with a descriptive name. 
+
+A good branch name would be (where issue #25 is the ticket you're working on): **25-annotations_to_tasks**
+
+```sh
+git checkout -b 25-annotations_to_tasks
+```
+
+### IDE configuration
+To know how configurate your development environment, see [IDE configuration](https://github.com/process-analytics/mxgraph-sketch-js/blob/master/docs/development/ide-configuration.md)
```
Comment (tbouffard) [inline CONTRIBUTING.md]: currently we don't have docs for this
```suggestion
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+
+The repository contains configuration for [Eclipse Theia](https://theia-ide.org/) and [gitpod.io](https://www.gitpod.io/).
```
Comment (tbouffard) [inline CONTRIBUTING.md:54]: :+1:

File: README.md [hunk 6]
```diff
@@ -1,10 +1,49 @@
...
+
+:sparkles: A BIG thanks to all our contributors :slightly_smiling_face:
+
+
+## 📃 License
+
+`mxgraph-sketch` is released under the [Apache 2.0](LICENSE) license. \
+Copyright &copy; 2020, Bonitasoft S.A.
+
+Thanks to [Roughjs](https://github.com/rough-stuff/rough) for the sketch style.
```
Comment (tbouffard) [inline README.md]:
```suggestion
Thanks to [Roughjs](https://github.com/rough-stuff/rough) (MIT License) for the sketch style.
```

File: CONTRIBUTING.md (hunk 4 above)
Comment (csouchet) [reply CONTRIBUTING.md]: It's a bad renaming. I added the link to bpmn-vizualization-js

Comment (tbouffard) [reply CONTRIBUTING.md]: I suggest to remove it as most of the explanation from bpmn-visualization-js do not apply here

Comment (csouchet) [reply CONTRIBUTING.md]: Done

**Review by tbouffard — APPROVED** [review]
LGTM

**Comment by tbouffard** [comment]
Add contribution guide, badges, screenshot.

**Pull request merged**